Formula/acmetool.rb

```diff
@@ -17,165 +16,23 @@ class Acmetool < Formula
-        :revision => "5ccada7d0a7ba9aeb5d3aca8d3501b4c2a509fec"
-  end
-
-  go_resource "golang.org/x/sys" do
-    url "https://go.googlesource.com/sys.git",
-        :revision => "2c42eef0765b9837fbdab12011af7830f55f88f0"
-  end
-
-  go_resource "golang.org/x/text" do
-    url "https://go.googlesource.com/text.git",
-        :revision => "e19ae1496984b1c655b8044a65c0300a3c878dd3"
-  end
-
-  go_resource "gopkg.in/alecthomas/kingpin.v2" do
-    url "https://gopkg.in/alecthomas/kingpin.v2.git",
-        :revision => "947dcec5ba9c011838740e680966fd7087a71d0d"
-  end
-
-  go_resource "gopkg.in/cheggaaa/pb.v1" do
-    url "https://gopkg.in/cheggaaa/pb.v1.git",
-        :revision => "43d64de27312b32812ca7e994fa0bb03ccf08fdf"
-  end
-
-  go_resource "gopkg.in/hlandau/configurable.v1" do
-    url "https://gopkg.in/hlandau/configurable.v1.git",
-        :revision => "41496864a1fe3e0fef2973f22372b755d2897402"
-  end
-
-  go_resource "gopkg.in/hlandau/easyconfig.v1" do
-    url "https://gopkg.in/hlandau/easyconfig.v1.git",
-        :revision => "7589cb96edce2f94f8c1e6eb261f8c2b06220fe7"
-  end
-
-  go_resource "gopkg.in/hlandau/service.v2" do
-    url "https://gopkg.in/hlandau/service.v2.git",
-        :revision => "b64b3467ebd16f64faec1640c25e318efc0c0d7b"
-  end
-
-  go_resource "gopkg.in/hlandau/svcutils.v1" do
-    url "https://gopkg.in/hlandau/svcutils.v1.git",
-        :revision => "c25dac49e50cbbcbef8c81b089f56156f4067729"
-  end
-
-  go_resource "gopkg.in/square/go-jose.v1" do
-    url "https://gopkg.in/square/go-jose.v1.git",
-        :revision => "aa2e30fdd1fe9dd3394119af66451ae790d50e0d"
-  end
-
-  go_resource "gopkg.in/tylerb/graceful.v1" do
-    url "https://gopkg.in/tylerb/graceful.v1.git",
-        :revision => "4654dfbb6ad53cb5e27f37d99b02e16c1872fbbb"
-  end
-
-  go_resource "gopkg.in/yaml.v2" do
-    url "https://gopkg.in/yaml.v2.git",
-        :revision => "d670f9405373e636a5a2765eea47fac0c9bc91a4"
+  patch do
+    url "https://github.com/CL-Jeremy/acmetool/commit/7b7ac8c75e936dcefff2c8b2903eb5c442db345f.patch?full_index=1"
+    sha256 "1cd4871cbb885abd360f9060dd660f8e678d1143a182f3bb63bddba84f6ec982"
   end
 
   def install
-    ENV["GOPATH"] = buildpath
-
-    (buildpath/"src/github.com/hlandau").mkpath
-    ln_sf buildpath, buildpath/"src/github.com/hlandau/acme"
-    Language::Go.stage_deps resources, buildpath/"src"
-
-    cd "cmd/acmetool" do
-      # https://github.com/hlandau/acme/blob/HEAD/_doc/PACKAGING-PATHS.md
-      ldflags = %W[
-        -X github.com/hlandau/acme/storage.RecommendedPath=#{var}/lib/acmetool
-        -X github.com/hlandau/acme/hooks.DefaultPath=#{lib}/hooks
-        -X github.com/hlandau/acme/responder.StandardWebrootPath=#{var}/run/acmetool/acme-challenge
-        #{Utils.safe_popen_read("#{buildpath}/src/github.com/hlandau/buildinfo/gen")}
-      ]
-      system "go", "build", "-o", bin/"acmetool", "-ldflags", ldflags.join(" ")
-    end
+    ENV["GO111MODULE"] = "on"
```

Comment:
I dont think you need this? with `go.mod` present, it would be automatically using go modules build, no?

Reply:
Correct. It's only needed if `GOPATH` is set and the repo resides in `$GOPATH/src/<full path of package>`, which it was in my last PR. I removed `ENV["GOPATH"]` from my last PR without thinking much about this. Thanks!